src/table.js

```diff
@@ -26,18 +26,18 @@ function Table(props) {
         props.onClickIcon(e.target.parentElement.parentElement.id, props.id);
     }
 
-    const isGNDVisionTable = () => {
-        if(props.table.categorytitle !== undefined && props.table.categorytitle === "Green New Deal vision") {
-            return true
-        }
-        else return false;
-    }
+    // If there's a category title, it is the GND Vision section
+    const isGNDVisionTable = (props.table.categorytitle) ? true : false;
```

Comment:
🎨 might name this just `hasCategoryTitle` or something to make it more generalizable, but nbd